### DISCUSSION TITLE
adding scollar frame

### DISCUSSION THREAD

**Discussion by ShahwaizRaza** [TomSchimansky/CustomTkinter#1858] [General]
I am adding a scroller into the frame I am reading your documentation
https://customtkinter.tomschimansky.com/documentation/widgets/scrollableframe#example-code

but it's not clear how to add a scroller frame this also said about "structure into a class"? dint understand
so please clarify what lines should I add to my code to achieve the desired result.

**Comment by ShahwaizRaza** [comment]
I added a scroller but the frame not expanding by its content which I place in it

**Comment by loafthecomputerphile** [comment]
> I added a scroller but the frame not expanding by its content which I place in it

have you tried using `sticky="NSWE` when using the .grid method also did you set your weights when configuring your grid columns and rows?

**Reply by ShahwaizRaza** [reply]
thanks for the reply, but I am using the pack() method because, in my mainframe, I placed more frame in it like parent frames and parent frames has child frames. so I want a scroller in the mainframe when the monitor screen size is small it should show a y-scroller. i hope you understand.

**Comment by Akascape** [comment]
@drewquashie Try this: https://github.com/Akascape/CTkXYFrame

**Comment by onlyforpeace** [comment]
@ShahwaizRaza you don't have to add ascroller on a scrollableframe, a scollable frame run like a frame, but the scrollbar is integrate to the frame...

**Comment by ShahwaizRaza** [comment]
I have added a stroller to the frame and it works

 my_frame = CTkScrollableFrame(app, fg_color="#1b1b1c", corner_radius=14)
 my_frame.pack( expand=True, fill="both",  padx=5, pady=5, anchor="center")
 my_frame.pack_propagate(False)
 my_frame.bind_all("<Button-4>", lambda e: my_frame._parent_canvas.yview("scroll", -1, "units"))
 my_frame.bind_all("<Button-5>", lambda e: my_frame._parent_canvas.yview("scroll", 1, "units"))
